src/hooks/useA11yNotification.js

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from "react";
+
+export function useA11yNotification({ isUrgent = false, isAriaLiveHandledOutside = false }) {
```

Comment:
would change it to isAlert and not isUrgent - code is pretty hard to understand when using this

for example - why is Toast.types.NEGATIVE urgent?